BSP/Simulator/Src/BSP_WDTimer.c

```diff
@@ -40,12 +40,11 @@ bool BSP_WDTimer_DidSystemReset(void){
  */
 void BSP_WDTimer_Start(void) {
     if (!initialized) {
-        Simulator_Log(LOG_ERROR, "Used watchdog timer without initialization!\n");
-        exit(-1);
+        Simulator_Log_Location(LOG_ERROR, "Used watchdog timer without initialization!\n");
+        Fault_BitMap = Fault_WDOG;
+        EnterFaultState();
```

Comment:
I think it would be nicer to have an assertOSError for the simulator (where we pass in the bitmap we want to set to and then enter fault state. But this is fine for now.